index.html

```diff
@@ -0,0 +1,38 @@
+<!-- Edit -->
+
+<!DOCTYPE html>
+<html lang="sv">
```

Comment:
You're saying the language of the document is Swedish here, but all the text on the page is in English.